inc/saitypes.h

```diff
@@ -677,6 +688,19 @@ typedef struct _sai_segment_list_t
     sai_ip6_t *list;
 } sai_segment_list_t;
 
+typedef struct _sai_ternary_field_t {
+    union {
+        sai_uint8_t u8;
+        sai_uint16_t u16;
+        sai_uint32_t u32;
+    } value;
+    union {
+        sai_uint8_t u8;
+        sai_uint16_t u16;
+        sai_uint32_t u32;
+    } mask;
+} sai_ternary_field_t;
+
```

Comment:
There is no type defined here so serialize method will not know which value should be serialized. why not reuse acl_field struct ?

Reply:
Also this could be just 2 uint32 for simplification